src/oh-my-posh.ts

```diff
@@ -0,0 +1,220 @@
+const completionSpec: Fig.Spec = {
+  name: "oh-my-posh",
+  description: "",
+  subcommands: [
+    {
+      name: "init",
+      description: "Initialize oh-my-posh for your shell",
+      subcommands: [
+        {
+          name: "fish",
+          description:
+            "Initialize the prompt for fish, example usage: 'oh-my-posh init fish'",
+        },
+        {
+          name: "bash",
+          description:
+            "Initialize the prompt for bash, example usage: 'oh-my-posh init bash'",
+        },
+        {
+          name: "zsh",
+          description:
+            "Initialize the prompt for zsh, example usage: 'oh-my-posh init zsh'",
+        },
+        {
+          name: "pwsh",
+          description:
+            "Initialize the prompt for pwsh core, example usage: 'oh-my-posh init pwsh'",
+        },
+        {
+          name: "powershell",
+          description:
+            "Initialize the prompt for powershell, example usage: 'oh-my-posh init powershell'",
+        },
+        {
+          name: "cmd",
+          description:
+            "Initialize the prompt for cmd, example usage: 'oh-my-posh init cmd'",
+        },
+      ],
+      options: [
+        {
+          name: ["--config", "-c"],
+          description:
+            "The config file to use, example usage: 'oh-my-posh init --config ~/.config.omp.json'",
+        },
+        {
+          name: ["--print", "-p"],
+          description: "Print the init script",
+        },
+      ],
+    },
+    {
+      name: "get",
+      description: "Get oh-my-posh values",
+      subcommands: [
+        {
+          name: "millis",
+          description:
+            "Get the current timestamp in milliseconds, example usage: 'oh-my-posh get millis'",
+        },
+        {
+          name: "shell",
+          description:
+            "Get the current shell, example usage: 'oh-my-posh get shell'",
+        },
+      ],
+    },
+    {
+      name: "debug",
+      description: "Debug oh-my-posh, example usage: 'oh-my-posh debug'",
+      options: [
+        {
+          name: ["--config", "-c"],
+          description:
+            "The config file to use, example usage: 'oh-my-posh debug --config ~/.config.omp.json'",
+        },
```

Comment:
Looks like this option takes an filepath argument!


```suggestion
        {
          name: ["--config", "-c"],
          description:
            "The config file to use, example usage: 'oh-my-posh debug --config ~/.config.omp.json'",
           args: {
              name: "config file",
              template: "filepath"
           },
        },
```

Reply:
@mschrage not necessarily. It can, but a URL is also valid.

Reply:
Added